go.mod

```diff
@@ -40,3 +40,5 @@ require (
 	google.golang.org/appengine v1.6.8 // indirect
 	google.golang.org/protobuf v1.31.0 // indirect
 )
+
+replace crawshaw.dev/jsonfile => github.com/sr/jsonfile v0.0.0-20240301210704-69e8a5b5b148
```

Comment:
Are you gonna PR this upstream? If not may as well just rename the module and depend directly on it

Reply:
Good question, I think I am leaning towards no since it's kind of a ~weird use case?

Reply:
Like, the "correct" solution would be to have the CLI create users through an API, but...